src/PokemonGymOwner.java

```diff
@@ -0,0 +1,27 @@
+import java.util.List;
+
+public class PokemonGymOwner {
```

Comment:
Je had hier een overerving van PokemonTriner kunnen toepassen. Nu heb je in allebij een `pokemons` en een `name`